src/init.cpp

```diff
@@ -479,11 +479,27 @@ std::string HelpMessage(HelpMessageMode mode)
 
 std::string LicenseInfo()
 {
+    std::string URL_SOURCE_CODE = "<https://github.com/bitcoin/bitcoin>";
+    std::string URL_WEBSITE_ABOUT = "<https://bitcoincore.org/en/about>";
+    std::string URL_WEBSITE_CONTRIB = "<https://bitcoincore.org/en/contribute>";
     // todo: remove urls from translations on next change
     return CopyrightHolders(strprintf(_("Copyright (C) %i-%i"), 2009, COPYRIGHT_YEAR) + " ") + "\n" +
            "\n" +
            _("This is experimental software.") + "\n" +
            "\n" +
+           strprintf(_("The source code is available from %s."),
+               URL_SOURCE_CODE) +
+           "\n" +
+           "\n" +
+           strprintf(_("Additional information about %s is available at %s."),
```

Comment:
Two %s can make the translation a bit hard in some languages where the order of words in the sentence is a bit different.